source/reference/minio-server/settings/notifications/amqp.rst

```diff
@@ -0,0 +1,358 @@
+.. _minio-server-envvar-bucket-notification-amqp:
+.. _minio-server-config-bucket-notification-amqp:
+
+==========================
+AMQP Notification Settings
+==========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+This page documents settings for configuring an AMQP service as a target for :ref:`Bucket Notifications <minio-bucket-notifications>`. 
+See :ref:`minio-bucket-notifications-publish-amqp` for a tutorial on using these settings.
+
+Multiple AMQP Targets
+---------------------
+
+You can specify multiple AMQP service endpoints by appending a unique identifier ``_ID`` for each set of related AMQP settings to the top level key. 
+
+Examples
+~~~~~~~~
+
+For example, the following commands set two distinct AMQP service endpoints as ``PRIMARY`` and ``SECONDARY`` respectively:
+
+.. tab-set::
+   
+   .. tab-item:: Environment Variables
+      :sync: envvar
+
+      .. code-block:: shell
+         :class: copyable
+      
+         set MINIO_NOTIFY_AMQP_ENABLE_PRIMARY="on"
+         set MINIO_NOTIFY_AMQP_URL_PRIMARY="amqp://user:password@amqp-endpoint.example.net:5672"
+      
+         set MINIO_NOTIFY_AMQP_ENABLE_SECONDARY="on"
+         set MINIO_NOTIFY_AMQP_URL_SECONDARY="amqp://user:password@amqp-endpoint.example.net:5672"
+   
+      For example, :envvar:`MINIO_NOTIFY_AMQP_ENABLE_PRIMARY <MINIO_NOTIFY_AMQP_ENABLE>` indicates the environment variable is associated to an AMQP service endpoint with ID of ``PRIMARY``.
+   
+   .. tab-item:: Configuration Settings
+      :sync: config
+
+      .. code-block:: shell
+
+         mc admin config set notify_amqp:primary \ 
+            url="user:password@amqp://amqp-endpoint.example.net:5672" [ARGUMENT=VALUE ...]
+
+         mc admin config set notify_amqp:secondary \
+            url="user:password@amqp://amqp-endpoint.example.net:5672" [ARGUMENT=VALUE ...]
+
+      Notice that for configuration settings, the unique identifier appends to ``amqp`` only, not to each individual argument.
+
+
+Settings
+--------
+
+Enable
```

Comment:
Hm. I'm guessing we have to do it this way to avoid the duplication of text the *entire* way down for each tab item?

Reply:
That is having an H2 for every single setting

Reply:
Also provides nice in-page navigation (on the right toc) to jump to a specific setting.